destream/decompressors/xz.py

```diff
@@ -1,10 +1,10 @@
 from destream import ExternalPipe
 
-__all__ = ['Unxz']
+__all__ = ["Unxz"]
 
 
 class Unxz(ExternalPipe):
-    _mimes = ['application/x-xz']
-    _extensions = ['xz']
-    _command = 'unxz -c'.split()
-    _compression = 'xz'
+    _mimes = ["application/x-xz"]
+    _extensions = ["xz"]
+    _command = "unxz -c".split()
+    _compression = "xz"
```

Comment:
Not sure what logic I used for the prefix with `_`. It might be worth revisiting this in a different ticket